root/app/auto-proxy.sh

```diff
@@ -100,7 +104,11 @@ DUDE
             done
             echo "}" >> "/etc/nginx/http.d/auto-proxy-${CONTAINER}.subdomain.conf"
         fi
+        if [ -z "${swag_address}" ]; then
+            swag_address="${CONTAINER}"
+        fi
         sed -i "s|<container_name>|${CONTAINER}|g" "/etc/nginx/http.d/auto-proxy-${CONTAINER}.subdomain.conf"
```

Comment:
Shouldn't this be using swag_address instead of CONTAINER?

Reply:
D'oh
Nice catch